vmdb/app/models/ems_refresh/parsers/openstack_common/orchestration_stacks.rb

```diff
@@ -100,16 +99,17 @@ def parse_stack_output(output, stack_id)
         end
 
         def parse_stack_resource(resource)
-          uid = resource['physical_resource_id']
+          # TODO(lsmola) expose physical_resource_id in Fog and replace all occurrences with
+          # resource.physical_resource_id
+          uid = resource.attributes['physical_resource_id']
```

Comment:
Please upgrade to fog 1.30. Now you can use `resource.physical_resource_id`. Same for a line 163 below.